frontstage/views/secure_messaging.py

```diff
@@ -79,6 +77,10 @@ def create_message(session):
     """Handles sending of new message"""
     if request.method == 'POST':
         party_id = session['party_id']
+        headers = {
+            "Authorization": request.cookies['authorization'],
+            "Content-Type" : "application/json"
```

Comment:
If you use the json keyword (instead of data) on requests.put, then it will do the json conversion and add the application/json header automatically.

Reply:
Made the change definitely a better way of doing it